Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -1222,42 +1246,73 @@ extension PackageInfoMapper {
                             package: packageNameAndInfo.key,
                             product: name,
                             packageInfos: packageInfos,
-                            targetDependencyToFramework: targetDependencyToFramework
+                            targetDependencyToFramework: targetDependencyToFramework,
+                            packageConditionDescription: condition
                         )
                     }
                 }
             }
         }
 
-        fileprivate static func fromTarget(name: String, targetDependencyToFramework: [String: Path]) -> [Self] {
+        fileprivate static func fromTarget(
+            name: String,
+            targetDependencyToFramework: [String: Path],
+            packageConditionDescription: PackageInfo.PackageConditionDescription?
```

Comment:
I would call the parameter `condition`
```suggestion
            condition packageConditionDescription: PackageInfo.PackageConditionDescription?
```